pywwt/qt_widget.py

```diff
@@ -127,7 +127,7 @@ def _run_js(self, js):
 
 class WWTQtWidget(BaseWWTWidget):
 
-    def __init__(self, block_until_ready=False):
+    def __init__(self, block_until_ready=False, size=None, **kwargs):
```

Comment:
What is the ``size=None`` for?

Reply:
I was following the code from the issue, which included `size` as an argument. I don't see it anywhere else so I'll take it out.